### PR TITLE
[ nrf-ble-driver] Updater to version 4.1.4

### PR DESCRIPTION
**Pull request by jornbh**
Upgrade nrf-ble-driver to version 4.1.4

The newer version fixes some performance issues on windows. 
(Doing a device Firmware Upgrade on Windows

### CODE REVIEW

File: ports/nrf-ble-driver/portfile.cmake [hunk 1]
```diff
@@ -26,7 +26,7 @@ set(ENV{PATH} "$ENV{PATH};${GIT_EXE_DIRPATH}")
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
-    OPTIONS -DDISABLE_EXAMPLES= -DDISABLE_TESTS= -DNRF_BLE_DRIVER_VERSION=4.1.2 -DCONNECTIVITY_VERSION=4.1.2
+    OPTIONS -DDISABLE_EXAMPLES= -DDISABLE_TESTS= -DNRF_BLE_DRIVER_VERSION=4.1.4 -DCONNECTIVITY_VERSION=4.1.4
 )
 
 vcpkg_install_cmake()
```
Comment (JonLiu1993) [inline ports/nrf-ble-driver/portfile.cmake:32]: @jornbh ,Thanks for your pr ,Please add vcpkg_fixup_pkgconfig() under vcpkg_install_cmake()

**Comment by JonLiu1993** [comment]
@jornbh ,Please run _./vcpkg x-add-version nrf-ble-driver --overwrite-version_ after you submit your changes, and then submit it again to solve the version problem

**Comment by PhoebeHui** [comment]
@jornbh, can you sign off the CLA?

**Comment by JonLiu1993** [comment]
@jornbh, can you sign off the CLA?

**Comment by PhoebeHui** [comment]
Pinging @JonLiu1993 for response. Is work still being done for this PR?

**Comment by JonLiu1993** [comment]
@jornbh ,I’m going to close this PR because it’s been a long time since any update has been made. If this is something you’d like to see merged and work is being done, please reopen.

**Pull request closed without merging**